indico/modules/events/abstracts/controllers/abstract_list.py

```diff
@@ -144,7 +171,8 @@ def _process(self):
             if tpl_components.get('hide_abstract'):
                 self.list_generator.flash_info_message(abstract)
             return jsonify_data(**tpl_components)
-        return jsonify_form(form, back=_('Cancel'), form_header_kwargs={'action': request.relative_url})
+        return jsonify_form(form, back=_('Cancel'), disabled_until_change=not self.abstract,
```

Comment:
At least when cloning as an invited abstract the form will always require some input (the intended submitter).. so this could be `disabled_until_change=(not self.abstract or is_invited)`